library/modules/bigip_policy_rule.py

```diff
@@ -84,6 +86,15 @@
       event:
         description:
           - Events on which actions such as reset can be triggered.
+          - With set_variable action, it is used for specifying action event such as request or response.
+        type: str
+      expression:
+        description:
+          - A tcl expression used with set_variable action.
```

Comment:
same here should be:  C(set_variable) action

Reply:
This comment is addressed now